release/tasks/cleanup-vm/cleanup-vm.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     tekton.dev/categories: Automation
     tekton.dev/tags: kubevirt
     tekton.dev/displayName: "KubeVirt cleanup VM"
-    tekton.dev/platforms: "linux/amd64"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/arm64"
```

Comment:
these changes has to go to manifests in `templates` folder and then run `make generate-yaml-tasks`